hbase_sql/src/test/java/com/splicemachine/derby/impl/sql/execute/operations/ExternalTableIT.java

```diff
@@ -246,6 +393,27 @@ public void testEmptyDirectory() throws  Exception{
     }
 
     @Test
+    public void testEmptyDirectoryAvro() throws  Exception{
+        String tablePath = getExternalResourceDirectory()+"empty_directory_avro";
+        new File(tablePath).mkdir();
+
+        methodWatcher.executeUpdate(String.format("create external table empty_directory_avro (col1 varchar(24), col2 varchar(24), col3 varchar(24))" +
+                " STORED AS AVRO LOCATION '%s'",tablePath));
+    }
+
+    @Test @Ignore
```

Comment:
this test passes when run individually, but when the IT is run as a whole, the incorrect error is thrown- LANG_OBJECT_ALREADY_EXISTS_IN_OBJECT instead of DIRECTORY_REQUIRED.